tests/PPM.Tests.ps1

```diff
@@ -99,6 +113,13 @@ Describe 'Test Suite' {
         $nsxManagerNode = $inputData.$domain.'NSX Manager Nodes'[0]
         $nsxManager = $inputData.$domain.'NSX Manager'[0]
         $nsxEdgeNode = $inputData.$domain.'NSX Edge Nodes'[0]
+        $nsxEdgeFqdn = $inputData.$domain.'NSX Edge Fqdn'[0]
+        $virtualCenter = $inputData.$domain.'Virtual Center'[0]
+        $ariaLifeCycleManagerFqdn = "xint-vrslcm01.rainpole.io"
+        $ariaOperations = "xint-vrops01a.rainpole.io"
+        $ariaOperationsLogs = "sfo-vrli01a.sfo.rainpole.io"
+        $ariaAutomation = "xint-vra01a.rainpole.io"
+        $workspaceOneAccess = "xint-wsa01a.rainpole.io"
```

Comment:
Can we move this to the JSON specification either under the management domain or it's own?